llpc/tool/llpcPipelineBuilder.h

```diff
@@ -113,6 +112,11 @@ class PipelineBuilder {
   // Prints pipeline dump hash code and filenames.
   void printPipelineInfo(Vkgc::PipelineBuildInfo buildInfo);
 
+protected:
```

Comment:
I'd leave it public like other methods in this class. This will make unit testing easier.